image/descriptor.go

```diff
@@ -23,25 +23,22 @@ import (
 	"strings"
 
 	"github.com/opencontainers/go-digest"
+	"github.com/opencontainers/image-spec/specs-go/v1"
 	"github.com/pkg/errors"
 )
 
-type descriptor struct {
-	MediaType string `json:"mediaType"`
-	Digest    string `json:"digest"`
-	Size      int64  `json:"size"`
-}
+type descriptor v1.Descriptor
```

Comment:
updated,I used to define this type because of this function, 

> func (d *descriptor) validate(w walker, mts []string) error {

and now I changed it to

> func ValidateDescriptor(d *v1.Descriptor, w walker, mts []string) error {

PTAL